examples/xdp/main.go

```diff
@@ -0,0 +1,83 @@
+//go:build linux
+// +build linux
+
+// This program demonstrates attaching an eBPF program to a network interface
+// with XDP (eXpress Data Path). The program parses the IPv4 source address
+// from packets and writes the packet count by IP to an LRU hash map.
+// The userspace program (Go code in this file) prints the contents
+// of the map to stdout every second.
+// It is possible to modify the XDP program to drop or redirect packets
+// as well -- give it a try!
+package main
```

Comment:
This example makes use of techniques only available in Kernel 5.7 and newer. I think it would be nice to make the reader/user of this example aware of this in the comment.

Reply:
good call, updated!